.devcontainer/devcontainer.json

```diff
@@ -32,7 +32,7 @@
     "forwardPorts": [4200, 9876],
 
     // Use 'postCreateCommand' to run commands after the container is created.
-    "initializeCommand": "git config -l > ${localWorkspaceFolder}/.devcontainer/.gitconfig.all && git config --local -l > ${localWorkspaceFolder}/.devcontainer/.gitconfig.local",
+    "initializeCommand": "git config -l > \"${localWorkspaceFolder}\"/.devcontainer/.gitconfig.all && git config --local -l > \"${localWorkspaceFolder}\"/.devcontainer/.gitconfig.local",
     // Use 'postCreateCommand' to run commands after the container is created.
     "postCreateCommand": "cd ${containerWorkspaceFolder}/.devcontainer && /bin/bash ./setup.sh",
     "postAttachCommand": "cd ${containerWorkspaceFolder}/.devcontainer && /bin/bash ./attach.sh",
```

Comment:
If the workspace folder name contains whitespace, `postCreateCommand` and `postAttachCommand` will fail. Modify it to the following could prevent this issue:
```json
{
    "postCreateCommand": "cd \"${containerWorkspaceFolder}\"/.devcontainer && /bin/bash ./setup.sh",
    "postAttachCommand": "cd \"${containerWorkspaceFolder}\"/.devcontainer && /bin/bash ./attach.sh",
}
```

Reply:
The new commit applied this ifx.